docker/activemq/activemq

```diff
@@ -0,0 +1,769 @@
+#!/bin/bash
+set -x
+
+### BEGIN INIT INFO
```

Comment:
Why do we need this file?  Is it modified over the regular ActiveMQ file?  If so, did you put some comments to identify our mods, so upgrading to some new version is easier?